gravitee-node-notifier/src/main/java/io/gravitee/node/notifier/trigger/NotificationTrigger.java

```diff
@@ -186,10 +194,7 @@ public void handle(Long event) {
                                                             error
                                                         )
                                                     )
-                                                    .doFinally(() ->
-                                                        //trigger a new attempt
-                                                        this.scheduleNextAttempt()
-                                                    )
+                                                    .doFinally(this::scheduleNextAttempt)
                                                     .subscribe();
```

Comment:
Probably out-of-scope here also but I can see several `subscribe()` calls here while I would expect a single `subscribe()` to the global chain.